src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html

```diff
@@ -9,9 +12,13 @@
       ></i>
     </div>
     <div class="col-xs-4 field-input">
+      <div ng-show="isLoading" class="loading-spinner">
```

Comment:
can there be a screenshot of the loading state in the PR

Reply:
sure, will add in!